cmd/shipperctl/ui/prompt_test.go

```diff
@@ -0,0 +1,67 @@
+package ui
+
+import (
+	"bytes"
+	"io"
+	"reflect"
+	"testing"
+)
+
+func TestAskForConfirmation(t *testing.T) {
+	tests := []struct {
+		Name            string
+		stdin           io.Reader
+		message         string
+		ExpectedConfirm bool
+		ExpectedErr     error
+	}{
+		{
+			Name:            "confirm with y",
+			stdin:           bytes.NewBufferString("y\n"),
+			message:         "",
+			ExpectedConfirm: true,
+			ExpectedErr:     nil,
+		},
+		{
+			Name:            "confirm with Y",
+			stdin:           bytes.NewBufferString("Y\n"),
+			message:         "",
+			ExpectedConfirm: true,
+			ExpectedErr:     nil,
+		},
+		{
+			Name:            "reject with n",
+			stdin:           bytes.NewBufferString("n\n"),
+			message:         "",
+			ExpectedConfirm: false,
+			ExpectedErr:     nil,
+		},
+		{
+			Name:            "reject with N",
+			stdin:           bytes.NewBufferString("N\n"),
+			message:         "",
+			ExpectedConfirm: false,
+			ExpectedErr:     nil,
+		},
+	}
```

Comment:
What about adding an extra test for "trying again when if there is an invalid char"?